index.bs

```diff
@@ -214,45 +215,68 @@ In other words, this attribute returns [=latest reading=]["quaternion"].
 
 ### OrientationSensor.populateMatrix() ### {#orientationsensor-populatematrix}
 
-The {{OrientationSensor/populateMatrix()}} method  populates the given buffer with WebGL-compatible rotation matrix
+The {{OrientationSensor/populateMatrix()}} method  populates the given object with rotation matrix
 which is converted from the value of [=latest reading=]["quaternion"] [[QUATCONV]], as shown below
 
 <img src="quaternion_to_rotation_matrix.png" style="display: block;margin: auto;" alt="Converting quaternion to rotation matrix." width="50%" height="50%">
 
 where:
 
- - W = cos(θ/2).
- - X = V<sub>x</sub> * sin(θ/2).
- - Y = V<sub>y</sub> * sin(θ/2).
- - Z = V<sub>z</sub> * sin(θ/2).
+ - W = cos(θ/2)
```

Comment:
Cant we make these lower case? That seems more common (yes I know that upper case is often used on graphs for the axises )